src/compiler/scala/tools/nsc/typechecker/ImportTracking.scala

```diff
@@ -167,15 +167,15 @@ trait ImportTracking { self: Analyzer =>
                   toEmit.foreach { case culled @ (selector, (_, _, _)) =>
                     if (selector != last) {
                       val index = selectors.indexWhere(_ == selector)
-                      val editPos = infoPos.withStart(selector.namePos).withEnd(selectors(index + 1).namePos)
+                      val editPos = infoPos.copyRange(start = selector.namePos, end = selectors(index + 1).namePos)
                       emit(culled, delete(editPos))
                     }
                     else {
                       // info.tree.pos.end is one char after rbrace
                       val prev = selectors.lastIndexWhere(remaining.contains(_))
                       val comma = content.indexWhere(_ == ',', from = selectors(prev).namePos)
-                      val commaPos = infoPos.withStart(comma).withEnd(selectors(prev + 1).namePos)
-                      val editPos = infoPos.withStart(selector.namePos).withEnd(info.tree.pos.end - 1)
+                      val commaPos = infoPos.copyRange(start = comma, end = selectors(prev + 1).namePos)
+                      val editPos = infoPos.copyRange(start = selector.namePos, end = info.tree.pos.end - 1)
```

Comment:
the port uses `Span` which felt heavyweight for some reason.

https://github.com/scala/scala3/blob/main/compiler/src/dotty/tools/dotc/transform/CheckUnused.scala#L701

but that API has trailing point: `def Span(start: Int, end: Int, point: Int): Span` with overloads